src/community/geofence-server/src/main/resources/GeoServerApplication.properties

```diff
@@ -41,4 +41,5 @@ GeofenceRulePage.access.nullValid=*
 GeofenceRulePage.ALLOW=ALLOW
 GeofenceRulePage.DENY=DENY
 GeofenceRulePage.LIMIT=LIMIT
-GeofenceRulePage.duplicate=An identical rule is already present. Duplicates are not allowed.
\ No newline at end of file
+GeofenceRulePage.duplicate=An identical rule is already present. Duplicates are not allowed.
+GeofenceRulePage.allowedArea=Allowed area
```

Comment:
"Allowed area (WKT)" is probably going to be clearer (at least until we have a visual editor for it)